ksql-engine/src/test/resources/query-validation-tests/min-group-by.json

```diff
@@ -33,6 +33,19 @@
         {"topic": "S2", "key": 0, "value": "0,-2147483647"}
       ]
     },
+    {
+      "name": "min integer group by with nulls",
+      "statements": [
+        "CREATE STREAM TEST (ID bigint, VALUE integer) WITH (kafka_topic='test_topic',value_format='JSON', key='ID');",
+        "CREATE TABLE S2 as SELECT id, min(value) as MAX FROM test group by id;"
+      ],
+      "inputs": [
+        {"topic": "test_topic", "key": 1, "value": {"id":  1, "value":  null}}
```

Comment:
Maybe add two extra tests where we have more than one row for input and one of them has null.